src/utils.js

```diff
@@ -46,14 +51,19 @@ class WayScriptClient {
         return url;
     }
  
-    executeRequest(method, url, payload) {
+    executeRequest(method, url, payload, key=null) {
```

Comment:
Instead of a specific arg for a key, could we pass in a map of headers to include in the request?